frontend/src/modules/EditZing/Components/EmailModal.tsx

```diff
@@ -50,14 +51,43 @@ export const EmailModal = ({
     ]
   }
 
+  const individualEmails = (students: string[]) => {
+    return [
+      'lscstudypartners@cornell.edu',
+      ...students.map((email: string) => email),
+    ]
+  }
+
+  /**
+   * promise that sends emails to each individual student.
+   *
+   * @throws error if there are any errors in sending the email
+   */
+  const sendIndividualEmails = async () => {
+    await Promise.all(
+      selectedStudents.map(() => {
+        const emailRcpts = individualEmails(selectedStudents)
```

Comment:
Does this send the right number of emails? Probably don't want to include all of the students' emails in the recipients list because they will see each other when they are meant to be emailed individually

Reply:
Ah yes, my fault on this.